hack/dind-cluster.sh

```diff
@@ -324,12 +326,26 @@ os::provision::install-cmds ${DEPLOYED_ROOT}"
 }
 
 function nodes-are-ready() {
-  # Skip the SDN node since nothing is intended to be scheduled on it.
-  local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
-KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep -v ${SDN_NODE_NAME} | grep -v NotReady | grep Ready | wc -l")
-  node_count=$(echo "${node_count}" | tr -d '\r')
-  test "${node_count}" -ge "${NODE_COUNT}"
+  local bin_path=$(os::provision::get-bin-output-path "${ORIGIN_ROOT}")
+  local kc=$(os::provision::get-admin-config ${CONFIG_ROOT})
+  read -d '' template <<'EOF'
+{{range $item := .items}}
+  {{if not .spec.unschedulable}}
+    {{range .status.conditions}}
+      {{if eq .type "Ready"}}
+        {{if eq .status "True"}}
+          {{printf "%s\\n" $item.metadata.name}}
+        {{end}}
+      {{end}}
+    {{end}}
+  {{end}}
+{{end}}
+EOF
+  # Remove formatting before use
+  template=$(echo "${template}" | tr -d '\n' | sed -e 's/} \+/}/g')
```

Comment:
Without checking, do you need to evaluate the new-line in the call to `tr` with `tr -d $'\n'`? Will the current `tr` call remove the newline in the template on line 337?

I approached this issue with the following code:
```sh
is_event_template=(               \
"{{with \$tags := .status.tags}}" \
    "{{range \$tag := \$tags}}"   \
        "{{\$tag.tag}} "          \
    "{{end}}"                     \
"{{end}}"                         \
)
is_event_template=$(IFS=""; echo "${is_event_template[*]}") # re-formats template for use

# wait until the last event that occurred on the imagestream was the successful pull of the latest image
wait_for_command 'oc get imagestream openldap --template="${is_event_template}" | grep latest' $((60*TIME_SEC))
```

Also, `go1.6` and higher versions allow for new syntax to ignore whitespace between `{{` markers. I think we need to be compatible on `1.5` for the tests for now, but later we can do away with these transformations and still have readable templates.